soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12856,7 +12871,7 @@ void func_8084E6D4(Player* this, GlobalContext* globalCtx) {
                 } else {
                     this->actor.colChkInfo.damage = 0;
                     func_80837C0C(globalCtx, this, 3, 0.0f, 0.0f, 0, 20);
-                    this->getItemId == GI_NONE;
+                    this->getItemId = GI_NONE;
```

Comment:
good catch here, curious as to what this was breaking

Reply:
Honestly I don't think it was breaking anything, I just knew it wasn't correct.